dev-manuals/test-plans/minigames/regex-game-frontend/plans/u-regex-game-1-game-start.md

```diff
@@ -0,0 +1,35 @@
+# Regex Game: Game Start (`u.regex-game-1`)
+
+Version: V1.0, 09.08.2022, initial test \
+Author: Leon Hofmeister
+
+## Description
+
+Tests that the game can be started.
+
+## Precondition
+
+The tester has a copy of the game.
+
+## Postcondition
+
+The game was started.
+
+## Typical procedure
+
+1. Press start
+2. The game starts
+3. The first regex is loaded (from the backend, once available)
+
+## Alternative procedures
+
+3.1. The backend is unavailable - a fallback regex is shown
```

Comment:
shouldn't there be a failure message?

Reply:
That was tricky: There is no backend yet. Hence I had to improvise on how to define it without "verklausulieren" myself.
Subsequently, there is no such message about a missing backend yet.
For now, I'd like not to merge tests that must always fail because the functionality isn't present yet.

Reply:
I don't think it is relevant for the test case whether our implementation fulfills it or not. The test case should describe the desired behaviour, not the current status. Therefore, the test case should not be altered with the progress of our implementation.